docs/tutorials/running_on_lfrqa.md

```diff
@@ -0,0 +1,292 @@
+# Preparing the LFRQA Dataset
+
+## Overview
+
+The **LFRQA dataset** was introduced in the paper [_LFRQA: Large-Scale Few-Shot Retrieval Question Answering_](https://arxiv.org/pdf/2407.13998). It features **1,404 science questions** (along with other categories) that have been human-annotated with answers. This tutorial walks through the process of setting up the dataset for use.
+
+## Download the Annotations
+
+First, we need to obtain the annotated dataset from the official repository:
+
+```bash
+# Clone the RAG-QA Arena repository
+git clone https://github.com/awslabs/rag-qa-arena
+
+# Create a new directory for the dataset
+mkdir data
+mkdir data/rag-qa-benchmarking
+
+# Move the science annotations file to our working directory
+mv rag-qa-arena/data/annotations_science_with_citation.jsonl ./data/rag-qa-benchmarking/
+
+# Clean up the repository folder
+rm -rf data/rag-qa-arena
+```
+
+## Download the Robust-QA Documents
+
+LFRQA is built upon **Robust-QA**, so we must download the relevant documents:
+
+```bash
+# Download the Lotte dataset, which includes the required documents
+curl https://downloads.cs.stanford.edu/nlp/data/colbert/colbertv2/lotte.tar.gz --output data/lotte.tar.gz
+
+# Extract the dataset
+tar -xvzf data/lotte.tar.gz
+
+# Move the science test collection to our dataset folder
+cp data/lotte/science/test/collection.tsv ./data/rag-qa-benchmarking/science_test_collection.tsv
+
+# Clean up unnecessary files
+rm data/lotte.tar.gz
+rm -rf data/lotte
+```
+
+For more details, refer to the original paper: [_LFRQA: Large-Scale Few-Shot Retrieval Question Answering_](https://arxiv.org/pdf/2407.13998).
+
+## Load the Data
+
+We now load both the questions and documents into a usable format:
+
+```python
+import os
+import pandas as pd
+
+# Load questions and answers dataset
+questions = pd.read_json(
+    os.path.join(
+        "data", "rag-qa-benchmarking", "annotations_science_with_citation.jsonl"
+    ),
```

Comment:
Let's just define the `"data", "rag-qa-benchmarking"` join once and use it across all snippets